tests/functional_tests/run_func_test.py

```diff
@@ -209,7 +215,7 @@ def test_writekey(self):
         self.input_files = self.files_to_input
         self.arguments = {
             "runType": "multi",
-            "calling_threshold": ".1",
+            "geno_call_threshold": ".1",
```

Comment:
@XingerTang Does it make sense to put value `.1` here (given that it will overwritten by 1/3)?